### DISCUSSION TITLE
Idea: Allow shortcodes in SiteTitle

### DISCUSSION THREAD

**Discussion by jptissot** [OrchardCMS/OrchardCore#7510] [Ideas]
This would allow us to have Multilingual titles in this single field by using the `[locale]` shortcode.

**Comment by hishamco** [comment]
I don't how the experience will look like, because there's no hint that title support `[locale]`

**Comment by sebastienros** [comment]
Shortcodes are for content editors, Liquid is for devs. Create a `locale` liquid tag. But might be better to ensure you can access the current culture from liquid, which might already be doable, then use standard liquid to select what to write.

But this makes me sad because next you'll want that for everything that needs localization, and this should be handled by dynamic localization.

**Reply by hishamco** [reply]
Exactly, dynamic localization will save us a lot of time from doing such stuff

**Reply by jptissot** [reply]
I will want it everywhere ;) Basically, I was trying to find a way to easily localize some settings that are displayed on the front end. Will dynamic localization add inline editors on fields ? Or will it be a different pages entirely ? Might be some UX considerations there.

**Reply by hishamco** [reply]
> Will dynamic localization add inline editors on fields ?

Basically dynamic data localization will applicable for all non static data such as menus, content types, content fields .. etc

**Comment by deanmarcussen** [comment]
What is `SiteTitle`?

Do  you mean `SiteName` or `PageTitle`? 

`PageTitle` already takes liquid, so @sebastienros suggestions fit
